### ISSUE TITLE
Show previous trades

### ISSUE THREAD

**Issue by jedmccaleb** [irisli/stellarterm#20]
There should be a list of your trade history somewhere.

**Comment by sacarlson** [comment]
yes this is another important feature that needs to be added like within account page.  another feature that if you don't add soon I will add to my branch of stellarterm.  At present my work around is I have my_wallet on another tab in my browser so I can view my trade history from that point.
